File: backend-api/reservation-api/src/main/java/com/hcmute/reservation/dto/override/OverrideLogResponse.java
package com.hcmute.reservation.dto.override;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder
public class OverrideLogResponse {
    private Long logId;
    private Long reservationId;
    private Long accountId;
    private String accountUsername;
    private String reason;
    private LocalDateTime createdAt;
}

File: desktop-staff/reservation-winforms/DTO/overrides/OverrideLogResponse.cs
using System;
using Newtonsoft.Json;

namespace reservation_winforms.DTO.overrides
{
    public class OverrideLogResponse
    {
        [JsonProperty("logId")]
        public long LogId { get; set; }

        [JsonProperty("reservationId")]
        public long ReservationId { get; set; }

        [JsonProperty("accountId")]
        public long AccountId { get; set; }

        [JsonProperty("accountUsername")]
        public string AccountUsername { get; set; }

        [JsonProperty("reason")]
        public string Reason { get; set; }

        [JsonProperty("createdAt")]
        public DateTime CreatedAt { get; set; }
    }
}